docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/upgrade-guides/2-0-0-upgrade-guide/index.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "2.0.0 upgrade guide"
+sidebar_position: -20
+---
+
+## Configuration
+
+BigQuery Loader 2.0.0 brings changes to the loading setup. It is no longer neccessary to configure and deploy three (in [1.X](/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/previous-versions/bigquery-loader-1.x/index.md) it was loader, repeater and mutator) independent applications in order to load your data to BigQuery.
+Starting from 2.0.0 only one appliction is needed, which naturally introduces some breaking changes to the configuration file structure.
+
+See the [configuration reference](/docs/pipeline-components-and-applications/loaders-storage-targets/bigquery-loader/configuration-reference/index.md) for all possible configuration parameters
+and the minimal [configuration samples](https://github.com/snowplow-incubator/snowplow-bigquery-loader/blob/v2/config) for each of supported cloud environments.
+
+## Events table format
+
+Starting from 2.0.0, BigQuery Loader no longer uses full schema version in column names for self-describing events and entities in `events` table. It uses only major schema version in the column name instead.
+
+This means all new self-describing events and entities will be loaded to 'major version' - oriented columns, whereas old 'full version' - oriented columns remain unchanged, are no longer used by the new loader, and have no direct effect on loading. If neccessary, old columns need to be handled/consolidated separately. //TODO how?
```

Comment:
The point about consolidating old columns is important.  You can ask our analytics engineers for help with phrasing this advice.

I _think_ the advice is..... if you use Snowplow's standard DBT models then don't worry it will all work ok automatically.  If you don't use Snowplow's DBT models then go figure it out for yourselves.